qfdmo/management/commands/export_displayedacteur.py

```diff
@@ -15,33 +16,41 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         self.stdout.write(f"Exporting Ressources, starting at {datetime.now()}")
+        target_file = datetime.now().strftime(
+            "exports/export_acteur_%Y%m%d_%H%M%S.xlsx"
+        )
 
-        if Path(TARGET_FILE).exists():
-            Path(TARGET_FILE).unlink()
+        with tempfile.NamedTemporaryFile(mode="w+b", suffix=".xlsx") as tmp_file:
 
-        try:
-            workbook = openpyxl.load_workbook(TARGET_FILE)
-            sheet = workbook.active
-        except FileNotFoundError:
-            workbook = openpyxl.Workbook()
-            sheet = workbook.active
+            try:
+                workbook = openpyxl.load_workbook("tmp.xlsx")
```

Comment:
```suggestion
                workbook = openpyxl.load_workbook(tmp_file.name)
```

Pas sûr que ce soit requis, mais il sert à quoi le `tmp.xlsx` ici. ?

Ou plutôt : veut-on vraiment écrire le fichier ligne 52, si on l'écrit juste après sur le s3 ?

Reply:
je pense que ça ne sert à rien mais il n'accepte pas `tmp_file.name` car c'est un chemin absolu.

Je suis d'accord avec toi que c'est crado :)

Reply:
J'ai fait une nouvelle proposition